crates/net/network/src/peers/reputation.rs

```diff
@@ -78,6 +78,7 @@ impl ReputationChangeWeights {
             ReputationChangeKind::Dropped => self.dropped.into(),
             ReputationChangeKind::Reset => DEFAULT_REPUTATION.into(),
             ReputationChangeKind::Other(val) => val.into(),
+            ReputationChangeKind::BadAnnouncement68 => DEFAULT_REPUTATION.into(), // todo: set val
```

Comment:
I think we can do 1 unit for this,
perhaps even add a new setting value for `bad_annoucements`?